service-loadbalancer/service_loadbalancer.go

```diff
@@ -268,6 +272,9 @@ func (lbc *loadBalancerController) getServices() (httpSvc []service, tcpSvc []se
 				Name: getServiceNameForLBRule(&s, servicePort.Port),
 				Ep:   ep,
 			}
+			if val, ok := s.Labels["lbHost"]; ok {
```

Comment:
please make this key more formal, and define it as a const. eg:
```
const lbHostKey =  "serviceloadbalancer/lb.host",
```

Eg how we do it for mirror pods: https://github.com/kubernetes/kubernetes/blob/e5670c104d1c8cdc5bd31f472b3ca806c5b2df35/pkg/api/pod_example.json#L12. becuase then we're free to define a `serviceloadbalancer/svcname.url` too.